### ISSUE TITLE
Gracefully fail of colordiff not installed

### ISSUE THREAD

**Issue by nk9** [NoahTheDuke/vim-just#33]
From [#23](https://github.com/NoahTheDuke/vim-just/issues/23#issuecomment-1479404918):

> I do think doing a check and failing more gracefully would be an improvement. Is that simple to achieve?

> Should be! We can check the output of `colordiff --version` and print something helpful.

**Comment by nk9** [comment]
I am also a Rust n00b. However this is more basic UNIX process management, and it sounds like a good approach to me.

**Comment by NoahTheDuke** [comment]
I think returning the Err is probably more clear, but yes that's the right approach: don't unwrap, pattern match on the Result and use the Ok value or handle the Err.

**Comment by nk9** [comment]
Yep, that would definitely have given me enough to go on. Thanks for your work on this @laniakea64!
